python_modules/dagster/dagster/_core/definitions/time_window_partitions.py

```diff
@@ -1032,6 +1062,28 @@ def is_basic_daily(self) -> bool:
     def is_basic_hourly(self) -> bool:
         return self.cron_schedule == "0 * * * *"
 
+    def get_fixed_minute_interval(self) -> Optional[int]:
```

Comment:
Would this make more sense in some sort of util module?

Reply:
Also a short docstring would be helpful - i.e. what exactly _is_ a fixed-minute interval?